packages/arb-token-bridge-ui/src/components/MainContent/MainContent.tsx

```diff
@@ -33,6 +36,8 @@ export function MainContent() {
   const [isArbitrumStatsVisible] =
     useLocalStorage<boolean>(statsLocalStorageKey)
 
+  const transactionHistoryProps = useTransactionHistory(address)
```

Comment:
Lifted the state up so it doesn't get reset when the transaction history unmounts.